certified-connectors/snowflake/apiProperties.json

```diff
@@ -5,7 +5,6 @@
         "type": "oauthSetting",
         "oAuthSettings": {
           "identityProvider": "aad",
-          "clientId": "54fbf434-4502-41d8-96a6-8e4fbe598e98",
           "scopes": [],
           "redirectMode": "Global",

```

Comment:
Please modify the redirectMode as "GlobalPerConnector, please refer the below link. And kindly remove the redirectUrl, team will guide you to update this url in your client app during certification. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment

Reply:
Hi @vmanoharas, implemented review comments, please verify.